### DISCUSSION TITLE
Adding styles.css in HTML head using shiny express

### DISCUSSION THREAD

**Discussion by martinlardi** [posit-dev/py-shiny#1096] [General]
I want to add `<link rel="stylesheet" type="text/css" href="styles.css">` inside of HTML head using shiny.express. How do I do it ? 
This is the code that I tried : 
`ui.tags.head(ui.tags.link(rel = "stylesheet", type = "text/css", href = "styles.css"))` 
It didn't work. it added link inside of HTML body.

**Comment by cpsievert** [comment]
This approach to adding styles assumes that `styles.css` is [served at that location to the app](https://shiny.posit.co/py/docs/ui-customize.html#serve-local-files). You might, instead, consider using [`ui.include_css()`](https://shiny.posit.co/py/docs/ui-customize.html#css-file)

**Comment by martinlardi** [comment]
Thanks ! `ui.include_css()` still add `<link href="styles.css">` inside of HTML body, not HTML head. However it works !
